### ISSUE TITLE
Limit history to specific folders

### ISSUE THREAD

**Issue by tit1** [antoniotejada/obsidian-edit-history#18]
allow users to select what folders they would like to keep a version history of. There are a lot of files in my vault that I do not require a version history of and currently, there is no way to limit which directory to watch or ignore.

**Comment by Epliskey** [comment]
Plugins tend to generate various additional files in the folder. It would be great to have a separate folder to organize them, keeping the main note files tidy.

**Comment by TAKAPlaza** [comment]
Yes, I would also like version history to be retained for specific files and folders.
